convex/dictionary.ts

```
@@ -0,0 +1,580 @@
+
+      if (existing) {
+        throw new Error("Entry already exists with this 'from' text");
+      }
+    }
+
+    // Validate 'to' text if provided
+    if (args.to && (args.to.length < 1 || args.to.length > 200)) {
+      throw new Error("'To' text must be between 1 and 200 characters");
+    }
+
+    const updates: any = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)